web/load.js

```diff
@@ -111,7 +111,7 @@
 
     extjsVersion = '6.2.0';
     fontAwesomeVersion = '4.7.0';
-    olVersion = '3.19.1';
+    olVersion = '3.20.0';
```

Comment:
Please update it in the "simple" folder as well. By the way, I don't remember if I showed it to you. It's basically a simple web interface (just map) without any extra dependencies.